src/view/navigation/routes.ts

```diff
@@ -160,6 +167,24 @@ export const routesInfo: RouteInfo[] = [
     component: LoanEntryDetailsPage,
     type: 'back',
   },
+  {
+    route: Route.ADD_PAYMENT_ENTRY,
+    name: 'Add Payment Entry',
+    component: AddPaymentEntryPage,
+    type: 'back',
+  },
+  {
+    route: Route.LOAN_ENTRY_DETAILS,
```

Comment:
This is why `Edit Entry` won't work, this should route to `EDIT_PAYMENT_ENTRY` not `LOAN_ENTRY_DETAILS`

Reply:
That was terrible, I can't believe I missed that. Updated.